projects/netgrif-components-core/src/lib/task-content/services/task-content.service.ts

```diff
@@ -296,6 +297,25 @@ export abstract class TaskContentService implements OnDestroy {
         });
     }
 
+    private updateReferencedField(chFields: ChangedFields, field: DataField<any>, frontendActions: Change): void {
+        if (this._fieldConverterService.resolveType(field) === FieldTypeResource.TASK_REF) {
+            this._taskDataReloadRequest$.next(frontendActions ? frontendActions : undefined);
+            return;
+        }
+        const updatedField = chFields[field.stringId];
+        Object.keys(updatedField).forEach(key => {
+            switch (key) {
```

Comment:
this switch feels very familiar. Is it not a duplicate of some other change resolution code elsewhere?

Reply:
This contains different logic than the similar switch in the same file.